### ISSUE TITLE
Auto mapping by channel name

### ISSUE THREAD

**Issue by w84no1** [xteve-project/xTeVe#367]
I am moving to xteve from TVMosaic. TVMosaic would auto map by channel name.  I don't want to have to manually map 250 channels.

**Comment by Deihmos** [comment]
You are better off using threadfin. Xteve is dead.

**Comment by MCTyler** [comment]
Will threadfin auto map 3rd party epg to m3u? @Deihmos 
automap occurs only in xTeVe when channel_id and epg channel name match, otherwise it's manual.

**Comment by KZkici** [comment]
tbh i am still having better luck with xteve than threadfin

**Comment by Deihmos** [comment]
Xteve is dead and hasn’t seen an update in years. Personally, I use Telly because it includes the channel numbers. I use it with Plex and don’t use xmltv but it is supported. 

Threadfin is under active development.

**Comment by dnsforge-repo** [comment]
Channel numbers are a feature of the client @Deihmos.  Fully featured in a client that supports it with xteve.

**Comment by Deihmos** [comment]
> Channel numbers are a feature of the client @Deihmos. Fully featured in a client that supports it with xteve.

xteve does not pass channel numbers from  the m3u playlist. It replaces it with its channel numbers beginning from 1. Telly is the only software I came across that allows you to use the channel numbers from the playlist.

**Comment by dnsforge-repo** [comment]
Well given channel numbers are provider specific for traditional tv, they are meaningless for iptv and different clients will handle them differently (or not at all). xTeVe keeps them ordered which is all that really matters!

**Comment by Deihmos** [comment]
I customize my m3u with channel numbers for a cable provider I had before IPTV, so when I add it to Plex, it auto matches the Plex guide which is superior. This actually works for both Telly and Threadfin. I decided to use Telly because it is simple and works.

**Comment by MCTyler** [comment]
Yes, adding channel numbers in m3u for Emby will have the same results but for most this is too much work. xTeVe only matches channel_id and channel_id in the EPG, otherwise it must be manually mapped in xTeVe.
